backend/src/Service/MailerService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Service;
+
+use PHPMailer\PHPMailer\PHPMailer;
+use Throwable;
+
+class MailerService
+{
+    private PHPMailer $mailer;
+
+    public function __construct()
+    {
+        $this->mailer = new PHPMailer(true);
+        $this->configure();
+    }
+
+    private function configure(): void
+    {
+        $this->mailer->isSMTP();
+        $this->mailer->Host = 'mailhog';
+        $this->mailer->Port = 1025;
+
+        $this->mailer->setFrom('agorium@email.com', 'Agorium');
+    }
+
+    public function sendWelcomeEmail(string $to, string $username): bool
+    {
+        try {
+            $this->mailer->addAddress($to, $username);
+
+            $this->mailer->isHTML(true);
+            $this->mailer->Subject = 'Welcome to Our App!';
```

Comment:
Vou deixar uma sugestão de texto, mas n vou barrar a PR. Se não quiser parar pra trocar tá de boa.

Welcome to Agorium!
Thank you for signing up **on** our app.

Reply:
Os emails penso que provavelmente vamos ter um layout, fica pra prox KKKK